content/providers/multiplatforms.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
+    "title": "Azure Quantum",
+    "image": null,
+    "description": "Qiskit provider for IonQ backends.",
+    "installation": "pip install \"azure-quantum[qiskit]\"",
+    "websiteCta": {
+      "label": "Website",
+      "url": "https://aka.ms/aq"
+    },
+    "docCta": {
```

Comment:
`docsCta`

Reply:
@techtolentino Corrected in https://github.com/Qiskit/qiskit.org/pull/2968/commits/237f7bc8a4991b49d4faf9df867edbc1505288fc Thanks for checking carefully!